test/next/src/app/layout.tsx

```diff
@@ -2,8 +2,6 @@ import { Inter } from 'next/font/google';
 
 const inter = Inter({ subsets: ['latin'] });
 
-export const dynamic = 'force-dynamic';
```

Comment:
Added it to the individual sub-pages instead so we don't force `force-dynamic` for `edge-config`, otherwise it prevents the reproduction of a specific bug there.